airbyte-api/src/main/openapi/config.yaml

```diff
@@ -4200,6 +4200,9 @@ components:
           format: int64
         succeeded:
           type: boolean
+        connectorConfigurationUpdated:
```

Comment:
nit: Could `connectorConfigurationUpdated` and `didUpdateConfiguration` both be the same name?  Perhaps `didUpdateConnectorConfiguration` works in both cases?

Reply:
I wanted to use `didUpdateConnectorConfiguration`, but PMD or some other check yelled at me because it wanted the get method on SynchronousJobMetadata to start with `is` :(

I can change the ConnectorJobOutput to be connectorConfigurationUpdated though

Reply:
Updated in [37b734d](https://github.com/airbytehq/airbyte/pull/21466/commits/37b734da2aa0ed63f324f3c55437378b0029508c)